Gaijinizer patch/Map199.txt

```diff
@@ -118,148 +124,163 @@ trigger .........」#MTLed
 　　人類を滅亡させる危険のある人物に
 　　この組織を預けることはできない………」
 > CONTEXT: Map199/events/1/pages/0/72/Dialogue
-*「I was thinking of leaving my chair to go with you ...
-To a person at risk of destroying humanity
-I can not deposit this organization .....」#MTLed
+*「I was thinking about giving up my position to you
+   someday, but I can't entrust this organization to
+   someone who may be plotting the downfall of
+   humanity...」#Ron
 > END STRING
 
 > BEGIN STRING
 報告に不手際があったことは謝ります。
 
 どうか、今一度お考え直しください。
 > CONTEXT: Map199/events/1/pages/0/91/Dialogue/other2
-I apologize for any inconvenience in the report.
-Please reconsider it once more.#MTLed
+I apologize for my ineptitude in my reports.
+
+Please reconsider what you're about to do.#Ron
 > END STRING
 
 > BEGIN STRING
 私はこの組織とこの世界の為に実行したまで。
 
 貴方を裏切るようなことなど……
 > CONTEXT: Map199/events/1/pages/0/95/Dialogue/other2
-Until I did it for this organization and this
-world. Things like betraying you ...#MTLed
+Everything I did was in service of this
+organization and the world.
+
+I would never betray you...#RonIffy
 > END STRING
 
 > BEGIN STRING
 ＊「…………我々の組織がどんなものなのか
 
 　　まさか忘れたわけではあるまい？」
 > CONTEXT: Map199/events/1/pages/0/100/Dialogue
-*「............ what is our organization. 
-I wonder if I forgot it?」#MTLed
+*「...Have you forgotten what kind of organization
+   this is?」#Ron
 > END STRING
 
 > BEGIN STRING
 ＊「『忠誠に背いた者には、死を』………」
 > CONTEXT: Map199/events/1/pages/0/104/Dialogue
-*「"For those who disbelief, death."」#MTLed
+*「"Disloyalty is death..."」#Ron
 > END STRING
 
 > BEGIN STRING
 そう………ですか………
 > CONTEXT: Map199/events/1/pages/0/106/Dialogue/other2
-Is that so………#MTLed
+Is... that so...#Ron
 > END STRING
 
 > BEGIN STRING
 残念です、大主教様。
 > CONTEXT: Map199/events/1/pages/0/110/Dialogue/other5
-Sorry, Lord Archbishop.#MTLed
+My apologies, Lord Archbishop.#Ron
 > END STRING
 
 > BEGIN STRING
 ＊「お…………お前は、\\N[1]………！？
 
 　　い、いや、違う………『これ』はっ……！！」
 > CONTEXT: Map199/events/1/pages/0/123/Dialogue
-*「Oh ............ You are \\N[1] .........! ?
-No, no, not......... "this" Ahh......! !」#MTLed
+*「Y-You're... \\N[1]...?!
+
+   N-No, it must "that"...!!」#Ron
 > END STRING
 
 > BEGIN STRING
 ＊「貴様…………！！！
 
 　　実の弟を、なんと冒涜的な………！！！」
 > CONTEXT: Map199/events/1/pages/0/127/Dialogue
-*「you…………!!! What a profane......!!!」#MTLed
+*「You...!!!
+
+   What blasphemous things did you do to your own
+   brother...?!!」#RonIffy
 > END STRING
 
 > BEGIN STRING
 『冒涜的』………？
 とんでもなく『合理的』でしょう……
 > CONTEXT: Map199/events/1/pages/0/131/Dialogue/other5
-"Blasphemy" .........? Outrageously "reasonable"
-...#MTLed
+Blasphemous...?
+
+This is perfectly rational...#Ron
 > END STRING
 
 > BEGIN STRING
 計画のために、無駄を省き、最適な解を導き出す……
 
 貴方を真似たのですよ、大主教様。
 > CONTEXT: Map199/events/1/pages/0/134/Dialogue/other5
-Eliminate waste and derive optimal solutions for
-planning...I copied you, Lord Archbishop.#MTLed
+For the sake of the plan, eliminate waste and
+arrive at the most optimal solution. I was merely
+imitating you, Lord Archbishop.#Ron
 > END STRING
```

Comment:
Some rewording:
>_**For the sake of my plan, I simply eliminated all
unnecessary waste to arrive at the most optimal
solution... I was simply following your example,
Lord Archbishop.**_